docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -71,27 +68,27 @@ remain lazy::
     >>> cube.has_lazy_data()
     True
 
-This is referred to as **lazy evaluation**.
+The process by which the operation is deferred until the result is requested is
+referred to as **lazy evaluation**.
 
 Certain operations, including regridding and plotting, can only be run on real data.
 Calling such operations on lazy data will automatically realise your lazy data.
 
 You can also realise (and so load into memory) your cube's lazy if you 'touch' the data.
```

Comment:
Oh! You were commenting literally on "your cube's lazy", which could be considered a personal statement regarding the cube's personal life... 

The word "data" is missing from that sentence, I'll change that right away.